tests/acceptance/organization-settings-profile-test.js

```diff
@@ -67,20 +67,14 @@ test("it allows editing of organization profile", (assert) => {
 
   let done = assert.async();
 
-  server.patch('/organizations/1', (db, request) => {
-    let params = JSON.parse(request.requestBody).data.attributes;
+  server.patch('/organizations/1', function(schema, request) {
+    let attrs = this.normalizedRequestAttrs();
 
-    assert.equal(params.name, 'Test User');
-    assert.equal(params.description, 'Lorem edit');
+    assert.equal(attrs.name, 'Test User');
+    assert.equal(attrs.description, 'Lorem edit');
     done();
 
-    return {
-      data: {
-        id: organization.id,
-        type: "organizations",
-        attributes: params
-      }
-    };
+    return this._getJsonApiDocForRequest(request, "organization");
```

Comment:
I'm not extremely excited with using `_getJsonApiDocForRequest `, since it's clearly intended to be private, but it's the only function I can find that gives us the structure we need, so it's still preferable to creating the structure manually. Alternatively, we could just return `JSON.parse(request.requestBody)`.